drivers/src/nrfx_pwm.c

```diff
@@ -92,6 +92,11 @@ static void configure_pins(nrfx_pwm_t const *        p_instance,
             bool inverted = output_pin &  NRFX_PWM_PIN_INVERTED;
             out_pins[i]   = output_pin & ~NRFX_PWM_PIN_INVERTED;
 
+            if (p_config->skip_gpio_cfg)
+            {
+                continue;
```

Comment:
I'd prefer to do not use continue as a point of exit from current context. 
```
    if (output_pin != NRFX_PWM_PIN_NOT_USED)
    {
        out_pins[i] = output_pin & ~NRFX_PWM_PIN_INVERTED;

        if (!p_config->skip_gpio_cfg)
        {
            nrf_gpio_pin_write(out_pins[i], output_pin & NRFX_PWM_PIN_INVERTED);
            nrf_gpio_cfg_output(out_pins[i]);
        }
    }
    else
    {
    ...
```

Reply:
I second that. Use of `continue` obscures the control-flow here.

Reply:
I use the suggested logic.